software/contrib/traffic.md

```diff
@@ -0,0 +1,52 @@
+# Traffic
+
+This script is inspired by [Jasmine & Olive Tree's Traffic](https://jasmineandolivetrees.com/products/traffic).
+Instead of 3 trigger inputs, this version only has 2.
+
+Traffic has 3 output channels (`cv1`-`cv3`), which output a CV signal when a trigger is received on either
+of the two inputs.  The value of the output signal depends on:
+1. which input the trigger was received on and
+2. the gains for that trigger on each channel.
+
+For example, suppose the gains for channel A are set to `[0.25, 0.6]`. Whenever a trigger on `din` is received,
+channel 1 (`cv1`) will output `MAX_VOLTAGE * 0.25 = 2.5V`. Whenever a trigger on `ain` in received, channel A
```

Comment:
Same as above comment for `ain`, and writing it for both will reinforce that the inputs are both being used as trigger inputs